app/src/sharedTest/java/org/oppia/app/topic/play/TopicPlayFragmentTest.kt

```diff
@@ -93,6 +89,17 @@ class TopicPlayFragmentTest {
     ).check(matches(hasDescendant(withText(containsString("3 Chapters")))))
   }
 
+  @Test
+  fun testTopicPlayFragment_loadFragmentWithTopicTestId0_storyProgress_isDisplayed() {
```

Comment:
Please also add a test for partial progress per the possible bug I outlined above (I suspect right now if you would observe 0% progress for partial completion due to the integer rounding error).

Reply:
Added partial and complete progress test cases.